andela/Ah-backend-groot
# #164069297 upgrade django 1.0 to django 2.1

**What does this PR do?**
* Upgrades codebase from Django 1.10 to Django 2.1

**Description of the tasks to be completed?**
* Update requirements using 
* Refactor URLs to a format that suites Django 2.1 

**How should this be manually tested?**

* Install project requirements using `pip install -r requirements.txt`
* Check for Django version

**What are the relevant pivot tracker stores?**
#164069297
